### PR TITLE
html-forms - James Yang

### PR DESCRIPTION
**Pull request by jamesdae**
https://user-images.githubusercontent.com/110082070/195179441-dcc7589d-326c-4861-878c-bca0bf2531c1.mov

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <input type="text" name="email" id="email"/>
+    <br>
+    <label for="reasons">Why is coding cool?</label><br>
+    <select name="reasons" id="reasons">
+      <option value="select an option">Select an option</option>
+      <option value="because it's interesting">Because it's interesting</option>
+      <option value="because it's fun">Because it's fun</option>
+      <option value="because it's powerful">Because it's powerful</option>
+    </select>
+    </p>
```
Comment (Zhiwutian) [inline html-forms/index.html:19]: remove this closing p tag since it doesn't have an opening tag

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <input type="radio" id="css" name="language" />
+    <label for="css">CSS</label>
+    <input type="radio" id="javascript" name="language" />
+    <label for="javascript">JavaScript</label>
+    <p>Would you like to sign up for more information?</p>
+    <input type="checkbox" name="signup" value="sign me up!"/> Sign me up!<br>
+    <label for="comments">Comments</label><br>
+    <textarea id="comments" cols="30" rows="10"></textarea> <br>
+    <input type="submit" name="submit" value="Submit"/>
+    </p>
```
Comment (Zhiwutian) [inline html-forms/index.html:32]: same as above

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small updates needed

**Comment by jamesdae** [comment]
https://user-images.githubusercontent.com/110082070/195192907-7e470a0b-1e5c-4c3c-a0a9-61a5f83969f2.mov

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    </select>
+    <p>Favorite Language (choose one)</p>
+    <input type="radio" id="html" name="language"/>
+    <label for="html">HTML</label>
+    <input type="radio" id="css" name="language" />
+    <label for="css">CSS</label>
+    <input type="radio" id="javascript" name="language" />
+    <label for="javascript">JavaScript</label>
+    <p>Would you like to sign up for more information?</p>
+    <input type="checkbox" name="signup" value="sign me up!"/> Sign me up!<br>
```
Comment (Zhiwutian) [inline html-forms/index.html:27]: make sure to include a label for this input

**Comment by jamesdae** [comment]
https://user-images.githubusercontent.com/110082070/195196793-d8803120-c842-4af3-8bcc-64be5c72f443.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**